### PR TITLE
[excel] (Requirement Set) Adding infomation for 1.9 release

### PR DESCRIPTION
**Pull request by AlexJerabek**
Staging this PR for when ExcelApi 1.9 rolls out in prod.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4ce66fb](https://github.com/OfficeDev/office-js-docs-pr/pull/918/commits/4ce66fb2e7290b1b41e31ec58390824873bb732d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904242338524268-918%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Irb1YcNfwkRVmfQBUR6lUKCs1%2F8uhfidXF%2BvRg9A1HQ%3D&st=2019-04-24T23%3A35%3A13Z&se=2019-05-25T23%3A40%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[2f3fdb8](https://github.com/OfficeDev/office-js-docs-pr/pull/918/commits/2f3fdb810ffbbaf2fef44ed23c3e7e23cac68664)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904242352380909-918%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ups4jt9KThYicFC1cR7S9G9VdSDe9%2B%2Bg%2B2Dz5PK1Z90%3D&st=2019-04-24T23%3A48%3A59Z&se=2019-05-25T23%3A53%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by AlexJerabek** [comment]
Here's the related-PR: https://github.com/OfficeDev/Office-Js-Requirement-Sets/pull/38/files

File: docs/overview/office-add-in-availability.md [hunk 1]
```diff
@@ -190,6 +193,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.6</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.7</a><br>
         - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.8</a><br>
+        - <a href="/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets">ExcelApi 1.9</a><br>
```
Comment (kbrandl) [inline docs/overview/office-add-in-availability.md]: In this page ([Host and platform availability](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-in-availability?branch=AlexJ-Excel19)), I notice you didn't add `ExcelApi 1.9` to the list of requirement sets under `Office 2019 for Windows` and `Office 2019 for Mac` -- were these two omissions intentional?

Comment (AlexJerabek) [reply docs/overview/office-add-in-availability.md]: @kbrandl Yes, those omissions were intentional. Office 2019 shipped with the July fork, which included up to Excel 1.8. Thanks for checking though.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8d2e49b](https://github.com/OfficeDev/office-js-docs-pr/pull/918/commits/8d2e49b051a92dac41f90382ebc43c5fe6008346)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |
[docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-Excel19/docs/reference/requirement-sets/excel-api-requirement-sets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C29%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904291819070854-918%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Aoyq3AKUWNXuLQsws3ugdwy%2BDRY623SIOQycw15wPpU%3D&st=2019-04-29T18%3A15%3A22Z&se=2019-05-30T18%3A20%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**